test-files/cmd/glacial/serial/ServiceBuildingReplacementTest.sh

```diff
@@ -3,8 +3,9 @@ set -e
 
 # Wholesale Building Replacement Workflow
 #
-# This test is lenient regarding the AOI, in that it will modify some features in the ref data that lie just outside of it. This workflow 
-# could work for other polygon or point data types but only buildings have been attempted so far.
+# This test is lenient regarding the AOI, in that it may modify some features in the ref data that lie just outside of the AOI. Secondary 
+# features crossing the AOI may be included in the output or conflated with reference features. This workflow should work other polygon 
+# data types, but only buildings have been tested with it so far.
 
 TEST_NAME=ServiceBuildingReplacementTest
 IN_DIR=test-files/cmd/glacial/serial/$TEST_NAME
```

Comment:
Service*ReplacementTest.sh are the tests for the `changeset-derive-replacement` command